modules/dxp/apps/portal-search-tuning/portal-search-tuning-synonyms-web/src/main/java/com/liferay/portal/search/tuning/synonyms/web/internal/configuration/SynonymsConfiguration.java

```diff
@@ -20,7 +20,7 @@
 public interface SynonymsConfiguration {
 
 	@Meta.AD(
-		deflt = "liferay_filter_synonym_en|liferay_filter_synonym_es",
+		deflt = "liferay_filter_synonym_de|liferay_filter_synonym_en|liferay_filter_synonym_es",
```

Comment:
I think we shouldn't add it to SynonymsConfiguration as deflt value since our goal is to add support only.

Reply:
@BryanEngler

Reply:
it should be default we discussed on daily